Feat/recover on crash (#487)
NordicSemiconductor/pc-nrfconnect-ppk: Changelog.md

## 4.3.0 - UNRELEASED

### Added

> -   Session recovery: option to recover a session after a crash or unexpected
>     shutdown.

Reviewer: ```suggestion
-   Session recovery feature, which lets you recover a session after a crash or an unexpected
    shutdown.
```

> -   Session recovery feature, which lets you recover a session after a crash or
>     an unexpected shutdown.

Reviewer: ```suggestion
-   Session recovery feature that lets you recover a session after a crash or
    an unexpected shutdown.
```
